src/MainWindow.cpp

```diff
@@ -89,61 +89,47 @@ MainWindow::MainWindow(BRect frame) : BWindow(frame, "", B_DOCUMENT_WINDOW, 0)
 	BMenuBar* bar = new BMenuBar("keybar");
 
 	BMenu* menu = new BMenu(B_TRANSLATE("Program"));
-
-	temp = B_TRANSLATE("Report a bug…");
-	menu->AddItem(new BMenuItem(temp.String(), new BMessage(M_REPORT_BUG)));
-	temp = B_TRANSLATE("Settings…");
-	menu->AddItem(new BMenuItem(temp.String(), new BMessage(M_SHOW_OPTIONS_WINDOW), ','));
-
+	menu->AddItem(new BMenuItem(B_TRANSLATE("Report a bug…"), new BMessage(M_REPORT_BUG)));
```

Comment:
Instead of "…" there's the constant B_UTF8_ELLIPSIS. There are quite a few of those to search&replace… :)